utils-testing-rx/build.gradle

```diff
@@ -0,0 +1,20 @@
+apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
+
+dependencies {
+    implementation "junit:junit:$versions.junit"
+
+    //Kotlin
+    implementation "org.jetbrains.kotlin:kotlin-stdlib:$versions.kotlin"
+
+    //Rx
+    implementation "io.reactivex.rxjava2:rxjava:$versions.rxjava"
+    implementation "io.reactivex.rxjava2:rxkotlin:$versions.rxkotlin"
+    implementation "io.reactivex.rxjava2:rxandroid:$versions.rxandroid"
+
+    // Mockito
+    implementation "org.mockito:mockito-core:$versions.mockito"
+    implementation "org.mockito:mockito-inline:$versions.mockito"
```

Comment:
No need for `testImplementation` instead? same goes for `junit`

Reply:
Not needed at all.

Reply:
junit is needed for building. There are no tests.